migrations/3_20190212095852_cards.js

```diff
@@ -2,7 +2,7 @@ exports.up = function(knex, Promise) {
   return knex.schema.createTable('cards', table => {
     table.increments();
     table.string('title').notNullable();
-    table.string('body', 1024).notNullable();
+    table.string('body', 1024);
```

Comment:
Have you already run this migration? Migration files should not be edited once they are created and run. The proper solution is to create a new migration file that fixes anything that was wrong with the old one, using tools like alterColumn or alterTable.

Reply:
created new migration to return this to notnullable in commit below